Dockerfile.pipelines

```diff
@@ -1,15 +1,8 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
```

Comment:
why this change?

Reply:
@tmbgreaves commented above about it, necessary for python versions